processing/src/main/java/org/apache/druid/guice/StartupInjectorBuilder.java

```diff
@@ -47,18 +48,15 @@ public StartupInjectorBuilder()
         new JacksonModule(),
         new ConfigModule(),
         new NullHandlingModule(),
+        new StorageConnectorModule(),
         new ExpressionProcessingModule(),
-        binder -> {
-          binder.bind(DruidSecondaryModule.class);
-        }
+        binder -> binder.bind(DruidSecondaryModule.class)
```

Comment:
Thanks for the cleanup. I guess I expected to find more stuff that would go here, but never did...